pykube/http.py

```diff
@@ -98,6 +99,23 @@ def send(self, request, **kwargs):
             pass
         elif "token" in config.user and config.user["token"]:
             request.headers["Authorization"] = "Bearer {}".format(config.user["token"])
+
+        elif "exec" in config.user:
+            exec_conf = config.user["exec"]
+
+            if exec_conf["apiVersion"] == "client.authentication.k8s.io/v1alpha1":
+                for env_var in exec_conf.get("env") or []:
+                    os.environ[env_var["name"]] = env_var["value"]
```

Comment:
it's better to pass environment variables explicitly to subprocess, see https://docs.python.org/3/library/subprocess.html

Reply:
thanks for pointing out. resolved it. please check